Gemfile

```diff
@@ -24,6 +24,7 @@ gem 'rails-controller-testing', '~> 0.1'
 gem 'govuk_ab_testing', '~> 2.0'
 gem 'htmlentities', '4.3.4'
 gem 'statsd-ruby', '1.3.0', require: 'statsd'
+gem 'sprockets-rails'
```

Comment:
I think it's worth adding a version constraint like "> 3.2" here to make it clear that we get a version that supports the integrity attribute.  As it is, the app already uses sprockets-rails so this entry might be redundant without the version constraint.  I'd probably be happy with dropping this from the Gemfile too TBH.